azure/iac/aks.rego

```diff
@@ -316,3 +316,111 @@ aks_aad_azure_rbac_metadata := {
     "Policy Help URL": "",
     "Resource Help URL": "https://docs.microsoft.com/en-us/azure/templates/microsoft.containerservice/managedclusters"
 }
+
+
+#
+# PR-AZR-0137-ARM
+#
+
+default aks_authorized_Ip = null
+
+azure_attribute_absence["aks_authorized_Ip"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.containerservice/managedclusters"
+    authorizedIPRange := resource.properties.apiServerAccessProfile.authorizedIPRanges[_]
```

Comment:
though its a array property in josn format checking like `not resource.properties.apiServerAccessProfile.authorizedIPRanges` should work as well.

Reply:
fixed